examples/stable-diffusion-1-5-img2img/examples.yaml

```diff
@@ -0,0 +1,4 @@
+- input:
+    inputs:
```

Comment:
Would be good to update the examples if possible, with something that works out of the box.